### PR TITLE
Add "Saturday" Track to !!rebecca.

### PR DESCRIPTION
**Pull request by Nessworthy**
Added Support for Rebecca Black - Saturday.. merge at your own risk.

### CODE REVIEW

**Comment by oldskool** [comment]
OMFG!!! I did not even know this existed. It's just as "good" as Friday... *facepalms*

Looks good to me for merging though :smile:

**Comment by Nessworthy** [comment]
I think the debate should really be whether it's wanted :p Friday is pretty iconic, as is the "Saturday" line from Friday. The track however, maybe not so much!

**Comment by gooh** [comment]
I couldn't care less for Saturday. IMO, it doesn't have any of Friday's charme. But if people want it, I am fine with it.

**Comment by kelunik** [comment]
I like the sentence for Saturday better than the video.

**Comment by gooh** [comment]
We can have both though. Just make the sentence into the link text.

**Comment by PeeHaa** [comment]
I think I could live with the current text with the addition of the saturday video link

File: src/Chat/Plugin/Rebecca.php [hunk 1]
```diff
@@ -28,9 +29,9 @@ private function getRebeccaLinkIfFriday(): string
         case 'Thursday':
```
Comment (staabm) [inline src/Chat/Plugin/Rebecca.php]: This method should be renamed

Comment (gooh) [inline src/Chat/Plugin/Rebecca.php]: yeah, probably sth like getResponse or calculateResponse…

**Comment by Nessworthy** [comment]
Ugh, really? Didn't expect it to not ff. What a pain.

**Comment by Nessworthy** [comment]
I'll close this off and re-do the thing tomorrow.

**Comment by Nessworthy** [comment]
☢

**Pull request closed without merging**